src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -783,6 +784,37 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
         return ResponseEntity.ok(stats);
     }
 
+    /**
+     * GET /courses/stats-for-management-overview :
+     *
+     * @param courseIds course Ids to get the stats for
+     * @param periodIndex period index for the user activity stats
+     * @return ResponseEntity with status
+     */
+    @GetMapping(value = "/courses/stats-for-management-overview")
+    @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
+    public ResponseEntity<List<CourseManagementOverviewCourseDTO>> getExerciseStatsForCourseOverview(@RequestParam(value = "courseIds[]") Long[] courseIds,
```

Comment:
```suggestion
    public ResponseEntity<List<CourseManagementOverviewCourseDTO>> getExerciseStatsForCourseOverview(@RequestParam("courseIds[]") Long[] courseIds,
```
Same here, maybe it is needed. I don't know honestly :)